desktop/apps/buy_art/stylesheets/index.styl

```diff
@@ -0,0 +1,350 @@
+@require '../../../components/stylus_lib'
+@require '../../../components/main_layout/stylesheets/buttons'
+@require '../../../components/main_layout/stylesheets/icons'
+@require './grid_styles'
+
+@font-face
+  font-family 'artsy-icons'
+  src url('/fonts/artsy-icons.eot?uo9ko')
+  src url('/fonts/artsy-icons.eot?#iefixuo9ko') format('embedded-opentype'),
+    url('/fonts/artsy-icons.woff2?uo9ko') format('woff2'),
+    url('/fonts/artsy-icons.ttf?uo9ko') format('truetype'),
+    url('/fonts/artsy-icons.woff?uo9ko') format('woff'),
+    url('/fonts/artsy-icons.svg?uo9ko#artsy-icons') format('svg')
+  font-weight normal
+  font-style normal
+
+global-reset()
+
+// General tag styles
+*,
+*:before,
+*:after
+  box-sizing border-box
+html[data-useragent*="Firefox"]
+  overflow-x hidden
+html, body
+  height 100%
+body
+  garamond()
+  font-size default-garamond-font-size
+  line-height 1.4em
+  text-rendering optimizeLegibility
+  -webkit-tap-highlight-color rgba(0, 0, 0, 0);
+a
+  color black
+  cursor pointer
+  -webkit-tap-highlight-color rgba(26, 26, 26, 0.301961);
+b, strong
+  font-weight bold
+em
+  font-style italic
+.small-caps
+  small-caps()
+input
+  garamond()
+  font-size default-font-size
+  appearance none
+  &[type='checkbox']
+    appearance checkbox
+  &[type='radio']
+    appearance radio
+  &:focus
+    outline none
+button, input
+  margin 0
+  outline none
+hr
+  position relative
+  height 1px
+  background-color gray-lighter-color
+  border none
+  margin 18px 0
+  clearfix()
+  &.is-invisible
+    background-color transparent
+sup
+  font-size 75%
+  line-height 1
+  vertical-align super
+sub
+  font-size 75%
+  line-height 1
+  vertical-align sub
+figure
+  margin 0
+
+#main-layout-container
+  margin-top (content-margin-top + main-header-height + welcome-header-height)
+
+// The 1250px max inner container
+.main-layout-container
+  max-width default-max-page-width
+  margin auto
+
+#blank-layout-container
+  height 100%
+
+// Scroll Frame styles
+#scroll-frame-spinner
+  position fixed
+  top 0
+  left 0
+  width 100%
+  height 100%
+  display none
+  z-index 10
+  background white
+.scroll-frame-loading #scroll-frame-spinner
+  display block
+.scroll-frame-open
+  overflow hidden
+
+#scripts
+  width 0
+  height 0
+
```

Comment:
Is everything above is from other stylesheets? No worries about bandwidth, just require what you need and then override whatever you need to!